go/vt/vtctl/workflow/materializer.go

```diff
@@ -494,10 +485,15 @@ func (mz *materializer) buildMaterializer() error {
 	if len(targetShards) == 0 {
 		return fmt.Errorf("no target shards specified for workflow %s ", ms.Workflow)
 	}
+	differentPVs := false
+	if sourceVSchema, err := mz.ts.GetVSchema(ctx, ms.SourceKeyspace); err == nil {
```

Comment:
Should we be handling this error? Otherwise on an error we assume that the vindexes are the same, while they may not be.